### ISSUE TITLE
Bug in IE 11 - Template tags get rendered as text

### ISSUE THREAD

**Issue by Markavian** [connected-web/web-component-js#4]
The browser version:
![image](https://cloud.githubusercontent.com/assets/1431385/8444660/46885e28-1f8b-11e5-9e72-ba8703e6a293.png)

The issue, see the **{{tags}}** at the bottom:
![image](https://cloud.githubusercontent.com/assets/1431385/8444662/58098e88-1f8b-11e5-812d-63538cbf5987.png)

Everything else appeared to render OK.

**Comment by Markavian** [comment]
Its a side effect of all those pesky divs at the bottom of the page - maybe as a CSS rule all `<template>` tags should have `display: none;` set, e.g.:

``` html
<style for="template">
template { 
  display: none;
}
</style>
```

![image](https://cloud.githubusercontent.com/assets/1431385/8444680/9b01f46e-1f8b-11e5-9b88-099214bd4df1.png)

**Comment by Markavian** [comment]
The browser version:
![image](https://cloud.githubusercontent.com/assets/1431385/8444775/dc401a54-1f8c-11e5-915e-cdbd1e6a2d43.png)

The issue, see the **{{tags}}** at the bottom:
![image](https://cloud.githubusercontent.com/assets/1431385/8444779/e877ff9e-1f8c-11e5-816d-0723c21c4b3a.png)

Everything else appeared to render OK.

**Comment by Markavian** [comment]
Apparently Firefox 16.0.2 has this problem too:
![image](https://cloud.githubusercontent.com/assets/1431385/8444833/a97a4b8e-1f8d-11e5-8270-d075582ef548.png)

And Firefox 17.0.1:
![image](https://cloud.githubusercontent.com/assets/1431385/8444847/dd6d3a82-1f8d-11e5-8a50-a61467cc297e.png)

But the issue was fixed in at least Firefox 38.0.5:
![image](https://cloud.githubusercontent.com/assets/1431385/8444857/0ac8dbd0-1f8e-11e5-8b34-bb061da13e42.png)
